x-pack/plugins/rule_registry/common/mapping_from_field_map.ts

```diff
@@ -11,7 +11,7 @@ import { FieldMap } from './field_map/types';
 
 export function mappingFromFieldMap(fieldMap: FieldMap): TypeMapping {
   const mappings = {
-    dynamic: 'strict' as const,
+    dynamic: true,
```

Comment:
Temporary workaround to unblock work, will not be merged. Seems to be newly introduced mapping issues related to the security solution rules that are being worked on.